src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 17px;
+  border-radius: $border-radius-default;
+  border: 1px solid $color-white-smoke;
+
+  &__image {
+        width: 160px;
+        height: 134px;
+  }
+
+  &__link {
+    margin-bottom: 40px;
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__name {
+    display: block;
+    color: $color-midnight-express;
+    font-size: 12px;
+    font-weight: 500;
+    text-decoration: none;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__code {
+    color: $color-comet;
+    font-size: 10px;
+    font-weight: 400;
+    margin-top: 0;
+    margin-bottom: 16px;
+    line-height: 14px;
+  }
+
+  &__reviews {
+      font-size: 10px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 14px;
+    }
+
+  &__rating {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__text {
+    color: $color-comet;
+    font-size: 12px;
+    font-weight: 400;
```

Comment:
It's a default value for this style
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/3b741a64-c8aa-4b92-b004-3c7c9018d2cf)

![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/97d582ab-6f76-498f-afeb-43497b66d460)

```suggestion
```